src/AlgoliaSearch.js

```diff
@@ -550,12 +553,27 @@ AlgoliaSearch.prototype = {
       })
     };
 
+    var JSONPParams = map(postObj.requests, function prepareJSONPParams(request, requestId) {
+      return requestId + '=' +
+        encodeURIComponent(
+          '/1/indexes/' + request.indexName + '?' +
```

Comment:
It is true that in most methods we do: `/1/indexes/ + encodeURIComponent(indexName)` so I guess we should do the same here.

Then how it's handled really is a question for the API server, I do not have more info here